source/renderer/app/components/wallet/wallet-create/WalletCreateSteps.scss

```diff
@@ -0,0 +1,8 @@
+.stepCounter {
+  font-family: var(--font-regular);
+  font-size: 12px;
+  line-height: 1.83;
+  margin-bottom: 5px;
+  text-align: center;
+  color: #5e6066;
```

Comment:
@daniloprates Please sort this CSS rule alphabetically.

Reply:
@yakovkaravelov done! Please, review it :)